src/http_server/cargo_dash_http_server.py

```diff
@@ -66,7 +105,22 @@ async def get_variable(request):
         }
 
     '''
-    pass
+    all_nodes = uncompress_nodes_information()
+    try:
+        node_id = int(request.match_info['id'])
+        var_name = request.match_info['var_name']
```

Comment:
is het een var? of een index?

Reply:
nope dit is een var_name staat ook in de docstring "var_name: name of the desired variable."